Dockerfile

```diff
@@ -16,9 +16,9 @@ MAINTAINER QuantConnect <contact@quantconnect.com>
 # RUN \
 #     cd /root/Lean && \
 #     sed -i 's/4.5/4.0/' Algorithm.VisualBasic/QuantConnect.Algorithm.VisualBasic.vbproj && \
-#     wget https://nuget.org/nuget.exe && \
+#     wget https://dist.nuget.org/win-x86-commandline/latest/nuget.exe && \
 #     mono nuget.exe restore QuantConnect.Lean.sln -NonInteractive && \
-#     msbuild /property:Configuration=Release && \
+#     msbuild QuantConnect.Lean.sln /property:Configuration=Release && \
```

Comment:
I'm surprised this wasn't working either? What was the error resolved by this change?

Reply:
I described the error in #3711, posting it here for reference.
```
MSBUILD : error MSB1011: Specify which project or solution file to use because this folder contains more than one project or solution file.
```